### ISSUE TITLE
Add support for TV shows

### ISSUE THREAD

**Issue by lmprice** [ankurd1/mdb#32]
Functionality is currently unreliable for pulling info for TV show episodes. It works for some episodes, but some will be detected incorrectly as a movie based on the title. 

Support for TV shows would be a very useful feature. guessit seems to have support for tv shows (mentioned in another issue).

**Comment by ankurd1** [comment]
Yeah, switching to guessit would be a better solution; instead of trying to fix this myself.
